cfme/tests/automate/test_service_dialog.py

```diff
@@ -482,3 +485,129 @@ def test_automate_methods_from_dynamic_dialog_should_run_as_per_designed(
         for ele_name in [ele_label, "label2", "label3"]:
             # Checking if automate method is not triggered after updating values of dialog widgets
             view.fields(ele_name).input.fill(fauxfactory.gen_alphanumeric())
+
+
+@pytest.fixture(scope="module")
+def custom_button(appliance):
+    """This fixture creates custom button for user object"""
+    def button(dialog):
+        collection = appliance.collections.button_groups
+
+        # Create custom button group
+        button_grp = collection.create(
+            text=fauxfactory.gen_alphanumeric(),
+            hover=fauxfactory.gen_alphanumeric(),
+            type=getattr(collection, "USER"),
+        )
+
+        # Create custom button
+        button = button_grp.buttons.create(
+            text=fauxfactory.gen_alphanumeric(),
+            hover=fauxfactory.gen_alphanumeric(),
+            dialog=dialog,
+            system="Request",
+            request="InspectMe",
+        )
+        return button_grp, button
+
+    return button
+
+
+def navigation_to_view(custom_button_group, button):
+    """This function helps to navigate on view comes after clicking on custom button"""
+    custom_button_group.item_select(button.text)
+    view = button.create_view(TextInputAutomateView, wait="10s")
+    return view.text_box2.read(), view
+
+
+@pytest.mark.tier(1)
+@pytest.mark.meta(automates=[1715396])
+@pytest.mark.customer_scenario
+@pytest.mark.parametrize(
+    "import_data",
+    [DatastoreImport("bz_1715396.zip", "bz_1715396", None)],
+    ids=["domain"]
+)
+@pytest.mark.parametrize("file_name", ["bz_1715396.yml"], ids=["dialog"])
+def test_dialog_element_values_passed_to_button(request, appliance, import_datastore, import_data,
+                                                import_dialog, custom_button, soft_assert):
+    """
+    Bugzilla:
+        1715396
+        1717501
+
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/8h
+        startsin: 5.10
+        casecomponent: Automate
+        testSteps:
+            1. Import the attached automate domain and dialog exports in BZ - 1715396. The automate
+               domain contains a method to populate the dynamic second element in the dialog.
+            2. Put 'Request: InspectMe' or Install object_walker automate domain from
+               https://github.com/pemcg/object_walker
+            3. Add a custom button to a VM (or any) object. The button should use the newly imported
+               BZ dialog, and should run object_walker when submitted (/System/Process: Request,
+               Message: create, Request: object_walker)
+            4. Click the custom button, and observe the dialog. The element 'Text Box 1' default
+               value is empty, the dynamic element 'Text Box 2' has been dynamically populated.
+               click on submit the button.
+            5. Repeat step 4, but type some some value(like "aa") in element 'Text Box 1'.
```

Comment:
typo : 5. Repeat step 4, but type some value(like "aa") in element 'Text Box 1'.